docs/UserGuide.md

```diff
@@ -246,18 +247,28 @@ Adds a new applicant to the list.
 Format: `add n/NAME hp/PHONE e/EMAIL p/POSITION`
 
 <box type="warning" header="**Caution**">
-    Duplicate applicants are not allowed. Two applicants are considered duplicates if they have the same phone number or email.
+    Duplicate applicants are not allowed. 
+    Two applicants are considered duplicates if they have the same phone number or email.
 </box>
 
 <box type="tip" header="**Note**">
-    As <code>NAME</code> can only contain alphanumeric characters and spaces, names containing hyphens <code>-</code> or slashes <code>/</code> are not allowed in this version of the app.  
+    As <code>NAME</code> can only contain alphanumeric characters and spaces, 
+    names containing hyphens <code>-</code> or slashes <code>/</code> are not allowed in this version of the app.  
```

Comment:
maybe we could add that other non alpha numeric characters in general are not allowed in the app 😃